src/php/Emitter.php

```diff
@@ -96,21 +78,21 @@ public function emitAndEval(Node $node): string
     public function eval(string $code)
     {
         $filename = tempnam(sys_get_temp_dir(), '__phel');
-        if ($filename) {
-            file_put_contents($filename, "<?php\n" . $code);
-            try {
-                $result = require $filename;
-            } catch (Throwable $e) {
-                throw $e;
-            }
-        } else {
+        if (!$filename) {
             throw new Exception("can not create temp file.");
         }
 
+        try {
+            file_put_contents($filename, "<?php\n" . $code);
+            $result = require $filename;
```

Comment:
@jenshaase Actually here, I would do the return directly. You don't really need to save the "require result" in a variable to return it later right after if no exception occurred.
```php
return require $filename;
```
The context is pretty strait forward here, and there is either `return something` or `throw $e`.
```php
try {
    file_put_contents($filename, "<?php\n" . $code);
    return require $filename;
} catch (Throwable $e) {
    throw $e;
}
```
Anyway, it's a minor thing. But just to let you know :)

Reply:
You are right. Missed this! Feel free to change this.